src/App.test.js

```diff
@@ -13,9 +13,15 @@ describe('App', () => {
     wrapper = shallow(<App />);
     expect(wrapper).toMatchSnapshot();
   });
+
   it('Should render without crashing', () => {
     const div = document.createElement("div");
-    ReactDOM.render(<App />, div);
+    ReactDOM.render(
+      <BrowserRouter>
```

Comment:
So smart to put the Browser Router here instead of in the component since we already have App wrapped in it!